docs/settings/settings.md

```diff
@@ -316,3 +316,26 @@ Default: `{"enable":true, "period":300}`
   "period": 300
 }
 ```
+
+## `storage-network`
+
+This setting allows you to assign network for Longhorn `storage-network` setting.
```

Comment:
I think this would be a better sentence if you feel that it makes sense.
```suggestion
This setting allows you to configure the Longhorn `storage-network` setting by assigning a network.
```